README.md

```diff
@@ -163,33 +156,24 @@ You can run some Tracim daemons too if you want those features:
     python3 daemons/mail_fetcher.py &
 
 You can now head to (if pushpin is correctly configured and use default port 7999)
-[http://127.0.0.1:7999](http://127.0.0.1:7999) and login with the admin account:
+[http://localhost:7999](http://localhost:7999) and login with the admin account:
 
  * user: `admin@admin.admin`
  * password: `admin@admin.admin`
 
-:warning: If this does not work, you can try to access [http://127.0.0.1:6543](http://127.0.0.1:6543). If it works, the issue is related to the configuration of pushpin.
+:warning: If this does not work, you can try to access [http://localhost:6543](http://localhost:6543). If it works, the issue is related to the configuration of pushpin.
 
-:warning: Using tracim without pushpin will mean no live message leading to unrefreshed frontend information.
+The full documentation about running the Tracim services with uWSGI and supervisor is available in the [Backend README](backend/README.md), sections `Running Tracim Backend Daemon`
+and `Running Tracim Backend WSGI APP`.
 
-### Upkeep
+### Upkeep
 
-If you use default "file" storage for session, new session file will be created but not removed, which mean lot of unneeded file.
-To mitigate this, you could remove file older than 8 day with :
+When the default "file" storage is used for session files you need to regularly remove old sessions files as they aren't removed automatically when the session expires (either of old age or when a user logs out). For example you can run this command every week:
 
     find . -type f -mtime +8 -print -exec rm {} \;
```

Comment:
```suggestion
    find <session.data_dir>/ -type f -mtime +8 -print -exec rm {} \;
```

Reply:
This is an example, so we need to make sure user not delete wrong file on his system.
We have an example here https://github.com/tracim/tracim/blob/maintenance/3.4/backend/doc/setting.md#delete-all-existing-sessions-file-storage  **but** I seen also a bad example here https://github.com/tracim/tracim/blob/maintenance/3.4/backend/doc/setting.md#file-storage-configuration-default

Reply:
I'm not sure the second example is bad as it aims to remove all session files (for example if the session parameters are changed it is recommended). What it's missing is the context where it is useful.

Reply:
In fact the second example has the context where it is useful (warning sign some lines above).

Reply:
After some discussion last time, we have decided to show example without risk of deleting wrong file. This is the reason why I purpose to remove `.` to other thing.

If its not necessary here, thats ok but I think its risky to put command like this in wrong path.

Reply:
If you think thats ok, do not chnage anything here

Reply:
I can add `-i` to the recommended command: this flag will make `rm` ask a confirmation for every file to be removed.

Reply:
Yes, it can be a good idea.